src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const allTr = document.getElementsByTagName('tr');
+const tbody = document.querySelector('tbody');
```

Comment:
```suggestion
const allTr = document.getElementsByTagName('tr');
const tbody = document.querySelector('tbody');
```
Let's use more meaningful names, so other developers can understand what they mean